.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules/
 .theia/
 .idea/
 
+packages/static-server
```

Comment:
This package was removed, so please get rid of it on your file system.

Reply:
I've removed it but it would have been nice to make it excluded in the .gitignore file for a while